platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-haliburton.init

```diff
@@ -16,9 +16,10 @@ start)
         echo -n "Setting up board... "
 
         depmod -a
```

Comment:
we should not have depmod -a in the init script, depmod -a should be in the postinst script.